modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/test/java/org/nuxeo/apidoc/core/test/TestBundleGroupExtractor.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Anahide Tchertchian
+ */
+package org.nuxeo.apidoc.core.test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.junit.Test;
+import org.nuxeo.apidoc.api.BundleGroup;
+import org.nuxeo.apidoc.api.BundleGroupExtractor;
+import org.nuxeo.apidoc.api.BundleInfo;
+import org.nuxeo.apidoc.introspection.BundleInfoImpl;
+
+/**
+ * @since 11.1
+ */
+public class TestBundleGroupExtractor {
+
+    class MockBundle {
```

Comment:
`protected static class`

Reply:
This did not feel necessary for an inner class in a (very) unit test: can you please elaborate on what it would bring here?

Reply:
Just to avoid warnings in tools that consider that the implicit package-private is wrong.

Reply:
Alright :+1: